cmake/Version.cmake

```diff
@@ -7,17 +7,21 @@
 #
 
 string(TIMESTAMP current_year "%Y")
-set(LY_VERSION_COPYRIGHT_YEAR ${current_year} CACHE STRING "Open 3D Engine's copyright year")
-set(LY_VERSION_STRING "0.0.0.0" CACHE STRING "Open 3D Engine's version")
-set(LY_VERSION_BUILD_NUMBER 0 CACHE STRING "Open 3D Engine's build number")
-set(LY_VERSION_ENGINE_NAME "o3de" CACHE STRING "Open 3D Engine's engine name")
+set(O3DE_COPYRIGHT_YEAR ${current_year} CACHE STRING "Open 3D Engine's copyright year")
+set(O3DE_VERSION_STRING "0.0.0" CACHE STRING "Open 3D Engine's version")
```

Comment:
Just going to do a quick check in our sandbox that Wix will support proper SemVers for packaging, as it appears to be still in progress: https://wixtoolset.org/docs/development/wips/burn-engine-semantic-versioning/

Reply:
It appears the version string is locked to a 4 octet format (xx.xx.xx.xx), but this doesn't affect the default install directory or editor version

Reply:
Thanks for checking! 😄